recipes-openxt/manager/xenmgr/xenmgr.initscript

```diff
@@ -17,90 +17,84 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 #
 
-# (use /bin/sh when watchdog stuff is removed)
-
-# Provides:		xenmgr
+### BEGIN INIT INFO
+# Provides:             xenmgr
+# Required-Start:       dbus-1 dbd
+# Required-Stop:
+# Default-Start:        5
+# Default-Stop:         0 1 6
+# Short-Description:    Starts OpenXT Xen manager.
+### END INIT INFO
+
+DESC="XenMgr"
+EXEC="/usr/bin/xenmgr"
+PIDFILE="/var/run/xenmgr.pid"
+OPTS=("--writepid=$PIDFILE" "--no-daemonize")
+
+. /etc/init.d/functions-selinux
```

Comment:
The commit message says `Use functions-dbus to get restore.` but I tyink you mean functions-selinux.

Reply:
Indeed, the commit message is wrong.